### PR TITLE
Awesome books: navigation & time stamp

### PR DESCRIPTION
**Pull request by yigitm**
We added the below features;

- Adding navigation bar with features: hiding and displaying contents
- Changing content in the main section on one page
- Adding date & time values to main content

Thanks for your time.

### CODE REVIEW

File: js/mainclass.js [hunk 1]
```diff
@@ -94,3 +99,36 @@ divCont.addEventListener('click', (e) => {
...
+  closeButton.addEventListener('click', () => {
+    navMenu.classList.remove('visible-mobile-nav');
+  });
+});
+/* eslint-disable no-undef */
+const { DateTime } = luxon;
+
+const dt = DateTime.now();
+const testHour = dt.toLocaleString(DateTime.DATETIME_MED);
+
```
Comment (rotimiazeez) [inline js/mainclass.js:133]: - [ ] Please make sure the live time and date in your local time zone is displayed below the nav bar😎

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team 👋, 
### Good job so far!🔥🔥

###There are some issues that you still need to work on to go to the next project but you are almost there!
> Kindly make the suggested changes to proceed to the Next stage🚀🚀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/mainclass.js (hunk 1 above)
Comment (iwillteixeira) [reply js/mainclass.js]: Done, insert setInterval function to update the time.

**Review by cyonii — CHANGES_REQUESTED** [review]
Hi team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._
- [x] The main functionalities of your app doesn't seem to be working at all, perhaps it has been broken by some recent changes, e.g, the navigation is not working, and it outputs this error (see image below) on the browser console. Please fix the issues.
Screen Shot 2021-11-25 at 1 03 47 AM



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by iwillteixeira** [comment]
Hi Cyonni, thanks for your review, but there is something wrong with it because I just tested here with the same pull request and didn't get any error, could you be more specific? our next review is the last one and I already fixed the previous review.

**Comment by iwillteixeira** [comment]
> # Hi Team 👋,
> ### Good job so far!🔥🔥
>  ###There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> > Kindly make the suggested changes to proceed to the Next stage🚀🚀
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello team 👋 , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**